Sprint-3/alarmclock/alarmclock.js

```diff
@@ -1,5 +1,63 @@
-function setAlarm() {}
+const heading = document.querySelector("#timeRemaining");
+const stopBtn = document.querySelector("#stop");
+heading.innerText = "Time Remaining: 00:00";
+const headerText = 'Remaining Time:'
 
+//==========formatTimeUnit============
+function formatTimeUnit(unit) {
+  return unit < 10 ? `0${unit}` : unit;
+}
+
+//===========setAlarm======================
+function setAlarm() {
+  const totalTimeInSeconds = parseInt(
+    document.querySelector("#alarmSet").value
+  );
+  setInitialMinutesAndSeconds(totalTimeInSeconds);
+  countDownEverySeconds(totalTimeInSeconds);
+}
+
+//=========formatAndDisplayTime=========================
+function formatAndDisplayTime(totalTimeInSeconds){
+  const totalMinutes = Math.floor(totalTimeInSeconds / 60);
+  const remainingSeconds = totalTimeInSeconds % 60;
+  const formattedMinutes = formatTimeUnit(totalMinutes)
+  const formattedSeconds = formatTimeUnit(remainingSeconds)
+  heading.innerText = `${headerText} ${formattedMinutes}:${formattedSeconds}`
+}
+
+//=============setInitialMinutesAndSeconds=======================
+function setInitialMinutesAndSeconds(totalTimeInSeconds) {
```

Comment:
I'm not sure how much value this function is bringing - what do you think is better about having this function, rather than just calling `formatAndDisplayTime(totalTimeInSeconds)` where it's currently called?

Reply:
I don't think it has much value in this context. We can just get rid of it and call ```formatAndDisplayTime(totalTimeInSeconds)``` where it is currently called